app/src/main/java/com/asfoundation/wallet/repository/OffChainTransactionsRepository.java

```diff
@@ -13,7 +13,7 @@ public OffChainTransactionsRepository(TransactionsApi api) {
   }
 
   public Single<WalletHistory> getTransactions(String wallet) {
-    return api.transactionHistory(wallet, "iap_offchain");
+    return api.transactionHistory(wallet, "offchain");
   }
 
   public interface TransactionsApi {
```

Comment:
Missing new query field on the transaction history request:
Please add version=(wallet_version_code) to the request